proxy/hdrs/MIME.cc

```diff
@@ -1197,6 +1197,27 @@ mime_hdr_length_get(MIMEHdrImpl *mh)
   return length;
 }
 
+int
+mime_hdr_net_length_get(MIMEHdrImpl *mh)
```

Comment:
I think it would be better if this function and ``mime_hdr_length_get`` shared an internal implementation, passing a parameter to determine whether to include internal headers.

Since you are studying this code, why is ``length`` initialized to 2? Is it to account for the ``\r\n``?